pandas/io/pickle.py

```diff
@@ -32,12 +44,21 @@ def read_pickle(path):
     ----------
     path : string
```

Comment:
Would be a nice enhancement (and welcome to work on this!), but let's leave that for another PR